src/circuits/tests.rs

```diff
@@ -427,7 +442,8 @@ mod test {
     // Passing an assets sum that is less than the liabilities sum should fail the lessThan constraint check
     #[test]
     fn test_is_not_less_than() {
```

Comment:
Can you add a test to check the case in which the second `assets_sum` is not less_than while the first one is?